libs/silly-wisher/README.md

```diff
@@ -3,3 +3,10 @@
 This library was generated with [Nx](https://nx.dev).
 
 This is a library for the Silly Wisher Assets used by Genshin Optimizer, with their permission to use for the April fools event.
+
+## Updating the splash and portrait art
+
+You can extract the assets from the Silly Wisher APK. I used
```

Comment:
I believe this is the way I was experimenting with it as well. Could you document any commands you needed to run, and the location of the assets once extracted, etc?

Also note down that images will need to be resized, and to what resolution